cmd/gitops/root/cmd.go

```diff
@@ -87,17 +90,19 @@ func RootCmd(client *resty.Client) *cobra.Command {
 
 	rootCmd.PersistentFlags().String("namespace", wego.DefaultNamespace, "The namespace scope for this operation")
 	rootCmd.PersistentFlags().StringVarP(&options.endpoint, "endpoint", "e", os.Getenv("WEAVE_GITOPS_ENTERPRISE_API_URL"), "The Weave GitOps Enterprise HTTP API endpoint")
+	rootCmd.PersistentFlags().StringVarP(&options.username, "username", "u", os.Getenv("WEAVE_GITOPS_USERNAME"), "The Weave GitOps Enterprise HTTP API endpoint")
+	rootCmd.PersistentFlags().StringVarP(&options.password, "password", "p", os.Getenv("WEAVE_GITOPS_PASSWORD"), "The Weave GitOps Enterprise HTTP API endpoint")
```

Comment:
Please remove `os.Getenv("WEAVE_GITOPS_PASSWORD")` as it shows my current password on the console.

Here's the leak, when display helps.
```
      --namespace string           The namespace scope for this operation (default "flux-system")
  -p, --password string            The Weave GitOps Enterprise HTTP API endpoint (default "12345678")
  -u, --username string            The Weave GitOps Enterprise HTTP API endpoint (default "wego-admin")
```

And the description would be "Password for ..."